machines/16x8/Makefile.machine.include

```diff
@@ -20,3 +20,5 @@ BSG_MACHINE_MEM_CFG                   = e_vcache_blocking_dramsim3_hbm2
 BSG_MACHINE_BRANCH_TRACE_EN           = 0
 
 BSG_MACHINE_HETERO_TYPE_VEC           = default:0
+
+BSG_MACHINE_IDEAL_NETWORK             = 0
```

Comment:
Question for @proftaylor: How do you feel about this naming? Is `IDEAL`, too *ideal*?